app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GuestRoomsController < ApplicationController
+    def create
+        @guest_room = GuestRoom.create(room_id: params[:room_id], guest_id: params[:id])
+        redirect_to "/guest/#{params[:id]}"
+    end
+end
```

Comment:
This looks great! I like that you're creating your GuestRoom right away with the info you pass in.